tests/security/test_kerberos.py

```diff
@@ -32,13 +33,26 @@ def setUp(self):
 
         if not configuration.conf.has_section("kerberos"):
             configuration.conf.add_section("kerberos")
-
-        configuration.conf.set("kerberos",
-                               "keytab",
+        configuration.conf.set("kerberos", "keytab",
                                os.environ['KRB5_KTNAME'])
+        keytab_from_cfg = configuration.conf.get("kerberos", "keytab")
+        self.args = Namespace(keytab=keytab_from_cfg, principal=None, pid=None,
+                              daemon=None, stdout=None, stderr=None, log_file=None)
 
     def test_renew_from_kt(self):
         """
         We expect no result, but a successful run. No more TypeError
         """
-        self.assertIsNone(renew_from_kt())
+        self.assertIsNone(renew_from_kt(args=self.args))
+
+    def test_args_from_cli(self):
+        """
+        We expect no result, but a run with sys.exit(1) because keytab not exist.
+        """
+        configuration.conf.set("kerberos", "keytab", "")
+        self.args.keytab = "test_keytab"
+
+        with self.assertRaises(SystemExit) as se:
+            renew_from_kt(args=self.args)
```

Comment:
What error is this hitting? Could you add some assertion of what error is logged too? (Probably use https://docs.python.org/3.6/library/unittest.html#unittest.TestCase.assertLogs via the backported `unittest2` which is already a dependency)

Reply:
it sys.exit(1) - https://github.com/apache/incubator-airflow/blob/3dc9e4021786e2c2416956f83a1f19ea1052ff3b/airflow/security/kerberos.py#L62 about assertLogs - okay, I will add it

Reply:
@ashb New in version 3.4. method added only in 3.4, at 2.7 will fail with 
`    with self.assertLogs(LoggingMixin().log) as log:
AttributeError: 'KerberosTest' object has no attribute 'assertLogs'`

Reply:
```
from unitteest2 import TestCase
```